examples/basic/main.tf

```diff
@@ -46,4 +46,12 @@ module "extenral_dns" {
   cluster_name                     = module.eks_cluster.eks_cluster_id
   cluster_identity_oidc_issuer     = module.eks_cluster.eks_cluster_identity_oidc_issuer
   cluster_identity_oidc_issuer_arn = module.eks_cluster.eks_cluster_identity_oidc_issuer_arn
+
+  settings = {
```

Comment:
please add more examples of different nested structure types like:
```
sources:
  # - crd
  - service
  - ingress
  # - contour-httpproxy

```
```
coredns:
  etcdTLS:
    enabled: false
```

Reply:
Could you check the other examples I just added?

Reply:
good enough